packages/run-utils/tests/main.js

```diff
@@ -0,0 +1,44 @@
+const test = require('ava')
+const { valid: validVersion } = require('semver')
+
+const run = require('..')
+
+test('Properties', t => {
+  t.is(typeof run, 'function')
+  t.is(typeof run.command, 'function')
+})
+
+test('Can run local binaries', async t => {
+  const { stdout } = await run('ava', ['--version'])
+  t.truthy(validVersion(stdout))
+})
+
+test('No arguments', async t => {
```

Comment:
Similar comment regarding test names, how about `should run command and return output`?

Reply:
:+1: 
I just improved all test titles :)